internal/testing/identity/dummy_identity.go

```diff
@@ -20,7 +19,7 @@ const (
 	DefaultAccountNumber = "1"
 )
 
-var xRhId = newIdentity(DefaultOrgId, ptr.To(DefaultAccountNumber))
```

Comment:
I guess this is fine as it won't have too much consequences, but I wonder:

We still use pointer to account_number in `models.Account`, but changing it here in tests.
I'd rather see it kept the same, tho migrating the account_number to string in prod might be bit tricky...
Currently we do not rely on the account_number anywhere IMO, so it's not a big deal, but what are your thoughts on having this difference?